core/evancore.vue.libs/src/components/user-callout/UserCallout.scss

```
@@ -0,0 +1,110 @@
+
+@import '~@evan.network/ui/src/style/utils';
+
+.callout {
+  position: fixed;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  width: 250px;
+  z-index: 1001;
+  transition: transform 0.3s ease-in-out;
+
+  &.active {
+    transform: translateY(calc(-100% + 61px));
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    // z-index: 1001;
+  }
+}
+
+/deep/ .callout .callout-button {
+  font-size: 12px;
+  font-weight: 600;
+
+  display: flex;
+  align-items: center;
+
+  height: 56px;
+  padding-left: 24px;
+  padding-right: 24px;
+
+  border: none;
+  outline: none;
+  background-color: white;
+
+  &, * {
+    color: cssVar('gray-900');
+  }
+
+  &:hover {
+    &, * {
+      color: cssVar('primary') !important;
+      border: 0;
+      background-color: inherit !important;
+    }
+  }
+}
```

I believe you don't need the "deep" modifier here.

```suggestion
.callout {
  position: fixed;
  display: flex;
  flex-direction: column;
  background-color: white;
  width: 250px;
  z-index: 1001;
  transition: transform 0.3s ease-in-out;

  &.active {
    transform: translateY(calc(-100% + 61px));
    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
    // z-index: 1001;
  }
  
  .callout-button {
  font-size: 12px;
  font-weight: 600;

  display: flex;
  align-items: center;

  height: 56px;
  padding-left: 24px;
  padding-right: 24px;

  border: none;
  outline: none;
  background-color: white;

  &, * {
    color: cssVar('gray-900');
  }

  &:hover {
    &, * {
      color: cssVar('primary') !important;
      border: 0;
      background-color: inherit !important;
    }
  }
}
```